packages/teleport/src/components/Authenticated/Authenticated.tsx

```diff
@@ -107,7 +121,7 @@ function startActivityChecker(ttl = 0) {
   return stop;
 }
 
-function isInactive(ttl = 0) {
-  const lastActive = localStorage.getLastActive();
+function isInactive(ttl = 0, getLastActive: () => number) {
```

Comment:
Can `getLastActive` be `lastActive: number` instead? As it's invoked immedIately I don't think it'll be stale by the time this function compares the values